packages/manager/modules/vps/src/cloud-database/vps-cloud-database.html

```diff
@@ -105,7 +105,7 @@ <h3 class="h1 mb-2" data-translate="vps_tab_cloud_database"></h3>
             <button
                 class="oui-button oui-button_primary"
                 type="button"
-                data-ui-sref="vps.detail.cloud-database.order"
+                data-ng-click="$ctrl.goToOrderCloudDatabase()"
```

Comment:
A button with a function that calls a url? It seems better to have a link with a `target` for accessibility reasons.

Reply:
Hello, thanks for the review. I changed the component as request. I also reduced the scope of the PR since your last review (only VPS)